city_scrapers/spiders/fortx_Fort_Worth_Isd.py

```diff
@@ -0,0 +1,79 @@
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class FortxFortWorthIsdSpider(CityScrapersSpider):
+    name = "fortx_Fort_Worth_Isd"
+    agency = "Fort Worth ISD Board"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.fwisd.org/board/board-of-education/board-calendar"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        # loop thru all dates that have events on them
+        for day in response.css(".fsStateHasEvents"):
+            meeting = Meeting(
+                title=day.css(".fsCalendarTitle::text").get(),
+                description="",
+                classification=BOARD,
+                start=self._parse_start(day),
+                end=self._parse_end(day),
+                all_day=False,
+                time_notes="",
+                location=self._parse_location(day),
+                links=self._parse_links(day),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _strip_timezone(self, string):
```

Comment:
To remove the timezone properly, I think you should follow these steps. 

First, parse datetime string to datetime object with tzinfo, there are 3 ways to parse:
```
from datetime import datetime
from dateutil.parser import parse

1. dt_with_tz = datetime.fromisoformat(datetime_string) 
2. dt_with_tz = datetime.strptime(datetime_string, "%Y-%m-%dT%H:%M:%S%z")
3. dt_with_tz = parse(datetime_string)
```

After that, you can replace the tzinfo with None.
```
dt_naive = dt_with_tz.replace(tzinfo=None)
```

Reply:
That's a good suggestion. Will add.